Source/bmalloc/bmalloc/Gigacage.cpp

```diff
@@ -73,7 +73,7 @@ struct PrimitiveDisableCallbacks : public StaticPerProcess<PrimitiveDisableCallb
     
     Vector<Gigacage::Callback> callbacks;
 };
-DECLARE_STATIC_PER_PROCESS_STORAGE(PrimitiveDisableCallbacks);
+DECLARE_STATIC_PER_PROCESS_STORAGE_WITH_LINKAGE(PrimitiveDisableCallbacks, BNOEXPORT);
```

Comment:
Ditto.

Reply:
Same as above: `PrimitiveDisableCallbacks` is an implementation detail of the primitive Gigacage APIs declared in "Gigacage.h". This change gives the storage object the same visibility as the class itself.